src/System.Windows.Forms/src/System/Windows/Forms/AccessibleObject.cs

```diff
@@ -2702,6 +2185,231 @@ private static IntPtr GetAddressOfVariantAtIndex(IntPtr variantArrayPtr, uint in
+            }
+
+            public string accValue
+            {
+                get => SafeGetValue(() => _systemIAccessible.accValue);
+                set => SafeExecute(() =>
+                {
+                    _systemIAccessible.accValue = value;
+                });
+            }
+
+            public string get_accValue(object childID)
+            {
+                return SafeGetValue(() => _systemIAccessible.get_accValue(childID));
+            }
+
+            public void set_accValue(object childID, string newValue)
+            {
+                SafeExecute(() => _systemIAccessible.set_accValue(childID, newValue));
+            }
+
+            public string accDescription => SafeGetValue(() => _systemIAccessible.accDescription);
+
+            public string get_accDescription(object childID)
+            {
+                return SafeGetValue(() => _systemIAccessible.get_accDescription(childID));
+            }
+
+            public object accRole => SafeGetValue(() => _systemIAccessible.accRole);
+
+            public object get_accRole(object childID)
+            {
+                return SafeGetValue(() => _systemIAccessible.get_accRole(childID));
+            }
+
+            public object accState => SafeGetValue(() => _systemIAccessible.accState);
+
+            public object get_accState(object childID)
+            {
+                return SafeGetValue(() => _systemIAccessible.get_accState(childID));
+            }
+
+            public string accHelp => SafeGetValue(() => _systemIAccessible.accHelp);
+
+            public string get_accHelp(object childID)
+            {
+                return SafeGetValue(() => _systemIAccessible.get_accHelp(childID));
+            }
+
+            public int get_accHelpTopic(out string pszHelpFile, object childID)
+            {
+                string helpFile = null;
+                int result = SafeGetStructValue(() => _systemIAccessible.get_accHelpTopic(out helpFile, childID), -1);
+                pszHelpFile = helpFile;
+                return result;
+            }
+
+            public string accKeyboardShortcut => SafeGetValue(() => _systemIAccessible.accKeyboardShortcut);
+
+            public string get_accKeyboardShortcut(object childID)
+            {
+                return SafeGetValue(() => _systemIAccessible.get_accKeyboardShortcut(childID));
+            }
+
+            public object accFocus => SafeGetValue(() => _systemIAccessible.accFocus);
+
+            public object accSelection => SafeGetValue(() => _systemIAccessible.accSelection);
+
+            public string accDefaultAction => SafeGetValue(() => _systemIAccessible.accDefaultAction);
+
+            public string get_accDefaultAction(object childID)
+            {
+                return SafeGetValue(() => _systemIAccessible.get_accDefaultAction(childID));
+            }
+
+            private protected TReturn SafeGetValue<TReturn>(Func<TReturn> getFunction) where TReturn : class
```

Comment:
If we have `Safe` in the name of the type, methods don't need to repeat it.
```suggestion
            private protected TReturn GetValue<TReturn>(Func<TReturn> getFunction) where TReturn : class
```

Reply:
Removed word `Safe` from the name of the methods.